[2025-05-28T08:53:56.178+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: csv_etl_pipeline.validate_csv manual__2025-05-28T08:53:23.544511+00:00 [queued]>
[2025-05-28T08:53:56.186+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: csv_etl_pipeline.validate_csv manual__2025-05-28T08:53:23.544511+00:00 [queued]>
[2025-05-28T08:53:56.187+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 4
[2025-05-28T08:53:56.199+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): validate_csv> on 2025-05-28 08:53:23.544511+00:00
[2025-05-28T08:53:56.209+0000] {standard_task_runner.py:57} INFO - Started process 14453 to run task
[2025-05-28T08:53:56.214+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'csv_etl_pipeline', 'validate_csv', 'manual__2025-05-28T08:53:23.544511+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/csv_etl_pipeline.py', '--cfg-path', '/tmp/tmp8s_09_24']
[2025-05-28T08:53:56.215+0000] {standard_task_runner.py:85} INFO - Job 8: Subtask validate_csv
[2025-05-28T08:53:56.268+0000] {task_command.py:416} INFO - Running <TaskInstance: csv_etl_pipeline.validate_csv manual__2025-05-28T08:53:23.544511+00:00 [running]> on host fe2c11ded3dc
[2025-05-28T08:53:56.361+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Sefanosok' AIRFLOW_CTX_DAG_ID='csv_etl_pipeline' AIRFLOW_CTX_TASK_ID='validate_csv' AIRFLOW_CTX_EXECUTION_DATE='2025-05-28T08:53:23.544511+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-28T08:53:23.544511+00:00'
[2025-05-28T08:53:56.825+0000] {logging_mixin.py:154} INFO - CSV Validation Results:
[2025-05-28T08:53:56.827+0000] {logging_mixin.py:154} INFO - - Rows: 100000
[2025-05-28T08:53:56.828+0000] {logging_mixin.py:154} INFO - - Columns: 9
[2025-05-28T08:53:56.829+0000] {logging_mixin.py:154} INFO - - Missing values: 0
[2025-05-28T08:53:56.864+0000] {python.py:194} INFO - Done. Returned value was: /opt/***/data/raw/data_20250528_085327.csv
[2025-05-28T08:53:56.904+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=csv_etl_pipeline, task_id=validate_csv, execution_date=20250528T085323, start_date=20250528T085356, end_date=20250528T085356
[2025-05-28T08:53:56.953+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-05-28T08:53:56.985+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
